Bottom: 06018e80e8f1ab2e975d92782ea074aa82651bda
Top:    359229a3f0184f6f9f824d38886e1925f2b48c68
Author: Jason Wang <jasowang@redhat.com>
Date:   2018-06-12 17:20:41 +0800

vhost_net: a 100% busy polling

Signed-off-by: Jason Wang <jasowang@redhat.com>


---

diff --git a/include/uapi/linux/vhost.h b/include/uapi/linux/vhost.h
index 3463db2..430ca28 100644
--- a/include/uapi/linux/vhost.h
+++ b/include/uapi/linux/vhost.h
@@ -174,6 +174,8 @@ struct vhost_memory {
 
 /* Use IOTLB message type V2 */
 #define VHOST_BACKEND_F_IOTLB_V2 0x1
+/* 100% busy polling vhost */
+#define VHOST_BACKEND_F_BUSY_POLLING 0x2
 
 #define VHOST_SET_BACKEND_FEATURES _IOW(VHOST_VIRTIO, 0x25, __u64)
 #define VHOST_GET_BACKEND_FEATURES _IOW(VHOST_VIRTIO, 0x26, __u64)
